spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -255,74 +256,90 @@ public static org.apache.iceberg.Table toIcebergTable(Table table) {
     return sparkTable.table();
   }
 
+  public static Transform[] toTransforms(Schema schema, List<PartitionField> fields) {
+    SpecTransformToSparkTransform visitor = new SpecTransformToSparkTransform(schema);
```

Comment:
I pulled the anonymous class below into `SpecTransformToSparkTransform` to reuse here.